From: NeilBrown <neilb@suse.com>
Subject: kabi FIX for SUNRPC: remove the maximum number of retries in
+ call_bind_status
Patch-mainline: never, kabi
References: git-fixes

Restore the removed field, even though it is never set.

Signed-off-by: NeilBrown <neilb@suse.com>
---
 include/linux/sunrpc/sched.h |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/include/linux/sunrpc/sched.h
+++ b/include/linux/sunrpc/sched.h
@@ -87,7 +87,8 @@ struct rpc_task {
 #endif
 	unsigned char		tk_priority : 2,/* Task priority */
 				tk_garb_retry : 2,
-				tk_cred_retry : 2;
+				tk_cred_retry : 2,
+				tk_rebind_retry : 2;
 };
 
 typedef void			(*rpc_action)(struct rpc_task *);
